source/agora/common/crypto/Key.d

```diff
@@ -120,8 +120,9 @@ public struct PublicKey
     }
 
     /// Uses Stellar's representation instead of hex
-    public string toString () const @trusted
+    public string toString () const @trusted nothrow
     {
+        scope (failure) assert(0);
```

Comment:
Unrelated change. I think you may want to rebase your PR first because this was merged a little while ago: https://github.com/bpfkorea/agora/pull/1125

Reply:
Thanks for the suggestions @AndrejMitrovic 
The problem with the rebase has been fixed and the changes have been applied.
Ready for review again.